src/components/Typography/Typography.js

```diff
@@ -83,7 +83,7 @@ const getTextStyle = (props) => {
 export const BaseText = styled.Text`
   font-family: ${appFont.regular};
   text-align-vertical: center;
-  color: ${themedColors.text};
+  color: ${({ color }) => color || themedColors.text};
   ${(props) => getTextStyle(props)}
```

Comment:
Hm, we cover bunch of colors from our color pallet in getTextStyle. One need just to add color name used in theme as a prop. Is that not enough?